common/menu.c

```diff
@@ -283,6 +332,14 @@ static void drawEntry(menuEntry_s* me, int off_x, int is_active) {
             snprintf(tmpstr, sizeof(tmpstr)-1, "%s: %s", textGetString(StrId_AppInfo_Version), me->version);
             DrawText(interuiregular14, start_x, start_y + 28 + 30 + 18 + 6 + 18, themeCurrent.textColor, tmpstr);
         }
+        
+        if (me->starred)
+        {
+            DrawText(largestar, start_x - 68, 160, themeCurrent.textColor, themeCurrent.labelStarOnText);
+        } else {
+            if (smallimg != theme_icon_small)//if (me->type != ENTRY_TYPE_THEME) <- why this crash?
```

Comment:
???

Reply:
The commented code crashed for me previously, unless I was running the wrong file or something. Fixed now, working fine.